miniupnpc/Makefile.mingw

```diff
@@ -88,6 +88,7 @@ pythonmodule:	libminiupnpc.a
 	$(PYTHON) setupmingw32.py install --skip-build
 	$(PYTHON) setupmingw32.py sdist
 	$(PYTHON) setupmingw32.py bdist_wheel --skip-build
+	7z a dist/*.whl miniupnpc.dll
```

Comment:
I don't think the AppVeyor builds are ok, check the wheels on PyPI.  They are lacking the `.dll`.  This, combined with the lack of published wheels for other platforms, is why we have been using our own not-on-PyPI publishing of miniupnpc wheels since we started using it a few years ago.

https://pypi.org/project/miniupnpc/2.2.8/#files
![image](https://github.com/user-attachments/assets/ca4aab73-82fc-4a75-933c-b7dc26f0fb77)

I'll try to find a non-7z solution using just Windows provided tooling hopefully.  Though, 7z seems like a simple dependency compared with setting up a full compilation environment on Windows.

Reply:
Why not use the static library ?

Reply:
I was just looking to add the file that is already expected, not change the linking method.  While I am certainly familiar with static vs. dynamic linking, I really don't know the specific tradeoffs around these options in Python packaging specifically.  Mostly just trying to get the published wheels working at all.  `:]`